### ISSUE TITLE
Suggestions

### ISSUE THREAD

**Issue by OmgImAlexis** [makkus/docker-seafile#12]
By using Caddy instead Nginx and/or Apache you can drop the whole needing to manage SSL certs part of this repo and let Caddy do it.

I noticed you're changing permissions using a command outside of the containers. It's best practice to do that inside the container on boot by setting an ENV in the container with the correct UUID and GUID. Is there a reason this isn't done this way?

**Comment by makkus** [comment]
Yeah, I had a go at using caddy a while ago, but couldn't get it to work in front of Seafile properly. Do you have a caddy config that works, incl. webdav, by any chance?

What permission change are you talking about exactly?

It's been a while since I touched this, I'm not sure how/whether all this still works with newer versions of Seafile. I had a go at refactoring it (develop branch), but in the end I got fed up. I find the docker-containers creation process just too messy for everything that is less than trivial, and I moved on to using lxc and ansible for my non-stateless-microservice-deployment needs. 

Happy to merge pull requests of course. Won't be testing them much though.
